### PR TITLE
Improve look and feel of community members

### PR DESCRIPTION
**Pull request by wd15**
Improve look and feel of community members and stop using Google forms
for the details. The members should only be other members of the group
that are involved in the project or attend workshops.

### CODE REVIEW

File: community.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+title: "Community Members"
+layout: default
+comment: >-
+  Links to people involved with the CHiMaD phase field
```
Comment (tkphd) [inline community.html]: Introductory text should set the scope and explain who gets in the list, and who does not. For example,
> The CHiMaD Phase Field working group is a growing community led by respected leaders in the field with valuable contributions from an international group of graduate students, postdocs, and researchers. This gallery provides biographical sketches and contact information for the active participants.

File: community.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  community. Please
+  email <a href="mailto:daniel.wheeler@nist.gov">daniel.wheeler@nist.gov</a>
```
Comment (tkphd) [inline community.html]: Instructions for how to get added should come at the end of the page, not up top, and should outline what information to include. For example:
> Is your sketch missing from the gallery? Please e-mail [Daniel Wheeler](mailto:daniel.wheeler@nist.gov) with the following details:
> - Your name, institutional affiliation, and contact information
> - A professional photograph or preferred avatar
> - A biographical sketch concisely summarizing your engagement with phase-field methods
>
> You will be contacted for any necessary clarification, and to let you know your profile has been added.

**Comment by wd15** [comment]
@tkphd, the suggested text is very nice, thanks

**Comment by tkphd** [comment]
Happy to help, of course.

**Comment by wd15** [comment]
@tkphd, I added your text, please approve if you like it

**Comment by tkphd** [comment]
@wd15, I would still put the instructions below the gallery. The avatars for Jim and Olle render somewhat compressed -- if there's a preferred aspect ratio or size for images, maybe add that the the guidelines as well. But these are stylistic, not substantial: go ahead and merge.

**Pull request merged**